test/libsolidity/ASTJSON/documentation.json

```diff
@@ -148,7 +148,6 @@
 					"id" : 14,
 					"implemented" : true,
 					"isConstructor" : false,
```

Comment:
Some other cleanup we could make: consistency between using the `is` prefix. Here we have `isConstructor` and `implemented`...

Reply:
Decision from the weekly meeting: only `isConstructor` uses the `is` prefix, lets remove it.

Also need to check if `fallback` is included.

Reply:
Now tracked in #4691 in 0.5.0 ``ready to be worked on``.